rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/mute/muterunner.kt

```diff
@@ -128,7 +132,7 @@ fun executeWithMutes(
     } catch (e: Throwable) {
         exceptions.add(e)
     }
-    return ExecutionResult(file, resolved.size, executed, failed, exceptions, result?.errors ?: emptyList())
+    return ExecutionResult(file, result?.root?.muteContexts?.size ?: 0, executed, failed, exceptions, result?.errors ?: emptyList())
```

Comment:
Are all of the mutes always resolved?

Reply:
The problem is with:
        val message =
            "$file - Total annotation: $resolved, executed ...
What I'd like to see here is the total nr. of annotation, while, in my opinion, the nr. of resolved annotation is confusing here. What do you suggest?